lang/main.json

```diff
@@ -275,6 +275,9 @@
         "alreadySharedVideoTitle": "Only one shared video is allowed at a time",
         "applicationWindow": "Application window",
         "authenticationRequired": "Authentication required",
+        "authenticationLogIn": "Log in",
```

Comment:
Don't we have all of these for web already?

Reply:
Can you check what the web dialog does pl?

Reply:
Updated.